src/tools/wix/Compiler.cs

```diff
@@ -3400,7 +3399,8 @@ private void ParseCustomActionElement(XElement node)
                 id = Identifier.Invalid;
             }
 
-            if (!explicitWin64 && (MsiInterop.MsidbCustomActionTypeVBScript == targetBits || MsiInterop.MsidbCustomActionTypeJScript == targetBits) && (Platform.IA64 == this.CurrentPlatform || Platform.X64 == this.CurrentPlatform))
+            // If we're a script CA and we're in a 64-bit package and our Architecture was "default", then set the 64-bit flag
```

Comment:
The WiX style is for comments to be complete sentences, so they should always be capitalized and have periods.